test/unit/anchore_engine/util/test_langpack.py

```diff
@@ -1,168 +1,300 @@
-import subprocess
-import unittest
+import pytest
 from anchore_engine.util.langpack import compare_versions
 
 enable_training = False
 
-class TestSemverVersionHandling(unittest.TestCase):
-    versions = [
```

Comment:
all of these, plus some new ones, are ported to fixtures+parametrize for a total of about 4 thousand tests